README.md

```diff
@@ -1,4 +1,8 @@
 # lib-fqm-query-processor
+Copyright (C) 2023 The Open Library Foundation
```

Comment:
2024?

Reply:
I figured 2023, since that's when the lib was donated to the OLF